src/main/scala/uk/gov/hmrc/ct/ct600/v3/retriever/CT600BoxRetriever.scala

```diff
@@ -34,6 +34,14 @@ trait CT600BoxRetriever extends ComputationsBoxRetriever with CT600DeclarationBo
 
   def b4(): B4 = B4(companyType())
 
+  def b5(): B5
```

Comment:
I would add these to the about this return retriever. From memory that's where they will end up.